### PR TITLE
Add simulation output

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

**Comment by rly** [comment]
@bendichter This is not my field. How specific is the term "simulation" for referring to the topic of this extension? As in, if I were to ask 100 neuroscientists what a "simulation" in neuroscience means, would this extension capture the vast majority of their answers? Does the extension handle simulations of both multi-compartment model neurons and networks of single-compartment, integrate-and-fire model neurons?

**Comment by bendichter** [comment]
@rly "simulation-output". The task is vastly simplified by this just storing the *output* of simulations. It's a general framework for storing compartment-wise continuous value recordings like membrane potential and calcium concentration, so it would be relevant for quite a few different simulations. Even integrate-and-fire neurons can fit into this because it's not about how the simulation is set up it's about the results. This may miss the mark for very abstract simulations where for instance you are modeling a cluster of neurons as a single value, or only modeling spikes and no compartment value at all, or for very anatomically detailed simulations that are concerned with e.g. channel kinetics, but I think it is likely to hit the mark for the majority of simulations that would be directly compared to neurophysiological data, as is the intention with this extension. I suppose if you wanted to make it more specific you could call it `network-simulation-output`

**Comment by rly** [comment]
Gotcha. Thanks for the explanation.

File: ndx-simulation-output/ndx-meta.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: my_simulation_output
```
Comment (rly) [inline ndx-simulation-output/ndx-meta.yaml]: please change name to: ndx-simulation-output

Comment (bendichter) [reply ndx-simulation-output/ndx-meta.yaml]: done

**Pull request merged**